backend/control_center/odl/views.py

```
@@ -598,11 +598,11 @@ def is_private_ip(ip):
                 print(f"Invalid data for classification: {e}")
                 results.append({"status": "error", "message": str(e)})
             except Exception as e:
-                logger.info(f"[ODL_CLASSIFY_AND_APPLY_POLICY] Error during ODL classification/policy application: {e}")
+                logger.error(f"[ODL_CLASSIFY_AND_APPLY_POLICY] Error during ODL classification/policy application: {e}")
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)